helm/values-uat.yaml

```diff
@@ -167,6 +167,9 @@ microservice-chart:
       { VISA:'https://assets.cdn.platform.pagopa.it/creditcard/visa.png', MASTERCARD:'https://assets.cdn.platform.pagopa.it/creditcard/mastercard.png', DINERS:'https://assets.cdn.platform.pagopa.it/creditcard/diners.png', AMEX:'https://assets.cdn.platform.pagopa.it/creditcard/amex.png', UNKNOWN:'https://assets.cdn.platform.pagopa.it/creditcard/generic.png', PAYPAL:'https://assets.cdn.platform.pagopa.it/apm/paypal.png', MAESTRO: 'https://assets.cdn.platform.pagopa.it/creditcard/maestro.png', MC: 'https://assets.cdn.platform.pagopa.it/creditcard/mastercard.png' }
 
     WALLET_ONBOARDING_PAYPAL_PSP_LIST: "BCITITMM,SIGPITM1XXX"
+    EXPIRATION_QUEUE_NAME: pagopa-u-weu-wallet-expiration-queue
+    EXPIRATION_QUEUE_TTL_SECONDS: "3600"
+    EXPIRATION_QUEUE_TIMEOUT_SECONDS_WALLET_CREATED: "900"
```

Comment:
add those variables to README too :)

Reply:
You read my mind updated here https://github.com/pagopa/pagopa-wallet-service/pull/148/commits/6f64f337f212802e5051e230de2fb7c5d34f1a68